src/main/java/org/sonar/plugins/stash/issue/CoverageIssue.java

```diff
@@ -0,0 +1,66 @@
+package org.sonar.plugins.stash.issue;
+
+import org.sonar.plugins.stash.StashPluginUtils;
+
+public class CoverageIssue extends Issue {
+
+  public static final String CODE_COVERAGE_ISSUE_KEY = "sonar.coverage.lowered";
+ 
+  private double uncoveredLines;
+  private double linesToCover;
+  
+  private double previousCoverage;
+  
+  public CoverageIssue(String severity, String path) {
+    super(CODE_COVERAGE_ISSUE_KEY, severity, path, (long) 0);
+  }
+
+  public double getUncoveredLines() {
+    return uncoveredLines;
+  }
+
+  public void setUncoveredLines(double uncoveredLines) {
+    this.uncoveredLines = uncoveredLines;
+  }
+
+  public double getLinesToCover() {
+    return linesToCover;
+  }
+
+  public void setLinesToCover(double linesToCover) {
+    this.linesToCover = linesToCover;
+  }
+  
+  public double getCoverage() {
+    double result = 0;
+    if (linesToCover != 0) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Equality tests should not be made with floating point values. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1244)